src/app/[team]/scrim/[scrimId]/map/[mapId]/page.tsx

```diff
@@ -127,6 +136,7 @@ export default async function MapDashboardPage({ params }: Props) {
         <div className="flex items-center justify-between space-y-2">
           <h2 className="text-3xl font-bold tracking-tight">
             {toTitleCase(mapName?.map_name ?? "Dashboard")}
+            <div className="text-lg">{map?.replayCode ?? ""}</div>
```

Comment:
I think this can be left off, I don't think this is necessary. We can skip this and leave it on the scrim dashboard page.